modeling.py

```diff
@@ -362,8 +371,12 @@ def dropout(input_tensor, dropout_prob):
 
 def layer_norm(input_tensor, name=None):
   """Run layer normalization on the last dimension of the tensor."""
-  return tf.contrib.layers.layer_norm(
-      inputs=input_tensor, begin_norm_axis=-1, begin_params_axis=-1, scope=name)
+  if FLAGS.use_fp16:
+    return fused_layer_norm.fused_layer_norm(
+      inputs=input_tensor, begin_norm_axis=-1, begin_params_axis=-1, scope=name,
+      use_fused_batch_norm=True)
```

Comment:
I'm not much of a TF person, so forgive me if this is a dumb question, but to what extent is this fused_layer_norm thing still helpful now that we're using XLA?

Reply:
I'm looking into whether I can get rid of this.